tests/qos/test_qos_sai.py

```diff
@@ -227,9 +228,21 @@ def testPfcStormWithSharedHeadroomOccupancy(
                                peer_info = peer_info)
         storm_hndle.deploy_pfc_gen()
 
-        # stop pfcwd
-        logger.info("--- Stopping Pfcwd ---")
-        duthost.command("pfcwd stop")
+        # check if pfcwd status is enabled before running the test
+        prev_state = duthost.shell('sonic-db-cli CONFIG_DB HGETALL "PFC_WD|{}"'.format(pfcwd_test_port))['stdout']
+        prev_poll_interval = duthost.shell('sonic-db-cli CONFIG_DB HGET "PFC_WD|GLOBAL" POLL_INTERVAL'.format(pfcwd_test_port))['stdout']
+
+        try:
+            prev_state = json.loads(prev_state)
+        except:
+            logging.debug("Couldn't parse {}".format(prev_state))
+            prev_state = {}
+
+        try:
+            prev_poll_interval = int(prev_poll_interval)
+        except:
+            logging.debug("Couldn't parse {}".format(prev_poll_interval))
+            prev_poll_interval = 0
```

Comment:
we only need to check if pfcwd is running before the test start: sonic-db-cli CONFIG_DB keys "PFC_WD*" should have some keys other than PFC_WD|GLOBAL